common/utilities.ts

```diff
@@ -172,6 +172,32 @@ export const getAuthAddress = async (headers) => {
   }
 }
 
+export const getUserStorageThreshold = async (headers) => {
```

Comment:
I'll add this on my TODO. I think we also want to get rid of the /stats call in pages since that one calculate storage against the content table each time and is running slow (not cache).